gatsby-node.js

```diff
@@ -0,0 +1,33 @@
+// From: https://gist.github.com/ivanoats/8d01d9e934fdc17bae9090147f1e799b
+
+const fs = require('fs');
+const sm = require('sitemap');
+
+function pagesToSitemap(pages) {
+  const urls = pages.map(p => {
+    if (p.path !== undefined) {
+      return {
+        url: p.path,
+        changefreq: 'daily',
+        priority: 0.7,
+      };
+    }
+  });
+  return urls.filter(u => u !== undefined);
+}
+
+function generateSitemap(pages) {
+  const sitemap = sm.createSitemap({
+    hostname: 'https://storybooks.js.org',
+    cacheTime: '60000',
+    urls: pagesToSitemap(pages),
+  });
+  fs.writeFileSync(`${__dirname}/public/sitemap.xml`, sitemap.toString());
```

Comment:
Should we be writing to `pages` rather than directly to disk? 

I don't really know anything about gatsby but it's kinda implied by this: https://github.com/gatsbyjs/gatsby#perform-additional-post-build-step